examples/vector_env/hello.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from maro.vector_env import VectorEnv
+from maro.simulator.scenarios.cim.common import Action, DecisionEvent
+
+if __name__ == "__main__":
+    with VectorEnv(batch_num=4, scenario="cim", topology="toy.5p_ssddd_l0.0", durations=100) as env:
+        for ep in range(2):
+            print("current episode:", ep)
+
+            metrics, decision_event, is_done = (None, None, False)
+
+            while not is_done:
+                action = None
+
+                # Usage:
+                # 1. Only push speicified (1st for this example) environment, leave others behind
+                # if decision_event:
+                #     env0_dec: DecisionEvent = decision_event[0]
+
+                #     # 1.1 After 1st environment is done, then others will push forward.
+                #     if env0_dec:
+                #         ss0 = env.snapshot_list["vessels"][env0_dec.tick:env0_dec.vessel_idx:"remaining_space"]
+                #         action = {0: Action(env0_dec.vessel_idx, env0_dec.port_idx, -env0_dec.action_scope.load)}
```

Comment:
yes, used dict to specify target environment, it will cause that only that environment will be push into next step, others will keep current state. use a list and single action will work as normal.